src/main/java/com/google/sps/data/Keywords.java

```diff
@@ -57,7 +57,9 @@
   public static String addKeywords(List<EntityAnnotation> blobAnalysis) throws IOException {
     List<String> labelList = new ArrayList<>();
     for (EntityAnnotation label : blobAnalysis) {
-      labelList.add(label.getDescription());
+      if (label.getScore() >= 0.75) {
```

Comment:
[Nit] I work try to avoid magic values in the code.  For some cutoff like 0.75 I would create a constant with a description name and maybe add a comment on how you decided on this value.

The same is true of 0.3 below.